src/components/Alert/README.md

```diff
@@ -0,0 +1,234 @@
+    <UIKit.Alert title="Outlined" message="Outlined theme" view="outlined" />
+</ExampleBlock>
+LANDING_BLOCK-->
+
+<!--GITHUB_BLOCK-->
+
+```
+<Alert title="Filled" message="Filled view" view="filled" />
+<Alert title="Outlined" message="Outlined theme" view="outlined" />
+```
+
+<!--/GITHUB_BLOCK-->
+
+### Layout
+
+`vertical` - used for direction of content if there is property `actions` with buttons. It shows buttons below text (
+used
+by default).
+
+`horizontal` - Used for direction of content if there is property `actions` with buttons. It shows buttons on the right
+side of text.
+
+<!--LANDING_BLOCK
+<ExampleBlock
+    code={`
+<Alert layout="vertical" title="Vertical" message="Vertical direction" actions={<Button>button</Button>} />
+<Alert layout="horizontal" title="Horizontal" message="Horizontal direction" actions={<Button>button</Button>} />
+`}>
+    <UIKit.Alert layout="vertical" title="Vertical" message="Vertical direction" actions={<UIKit.Button>button</UIKit.Button>} />
+    <UIKit.Alert layout="horizontal" title="Horizontal" message="Horizontal direction" actions={<UIKit.Button>button</UIKit.Button>} />
+</ExampleBlock>
+LANDING_BLOCK-->
+
+<!--GITHUB_BLOCK-->
+
+```tsx
+<Alert layout="vertical" title="Vertical" message="Vertical direction" actions={<Button>button</Button>}/>
+<Alert layout="horizontal" title="Horizontal" message="Horizontal direction" actions={<Button>button</Button>}/>
+```
+
+<!--/GITHUB_BLOCK-->
+
+### Corners
+
+`rounded` - used fot round corners of alert window (used by default)
+
+`square` - used for square corners of alert window.
+
+<!--LANDING_BLOCK
+<ExampleBlock
+    code={`
+<Alert title="Rounded" message="Rounded corners" corners="rounded"  />
+<Alert title="Square" message="Square corners" corners="square" />
+`}
+>
+    <UIKit.Alert title="Rounded" message="Rounded corners" corners="rounded"  />
+    <UIKit.Alert title="Square" message="Square corners" corners="square" />
+</ExampleBlock>
+LANDING_BLOCK-->
+
+<!--GITHUB_BLOCK-->
+
+```tsx
+<Alert title="Rounded" message="Rounded corners" corners="rounded"/>
+<Alert title="Square" message="Square corners" corners="square"/>
+```
+
+<!--/GITHUB_BLOCK-->
+
+## Alert title
+
+`title` - title of an alert. Override if by Alert.Title component.
+
+<!--LANDING_BLOCK
+<ExampleBlock
+    code={`
+<Alert title={<Alert.Title className={'some-class'} text="some text"></Alert.Title>} />
+`}
+>
+    <UIKit.Alert title={<UIKit.Alert.Title className={'some-class'} text="some text"></UIKit.Alert.Title>} />
+</ExampleBlock>
+LANDING_BLOCK-->
+
+<!--GITHUB_BLOCK-->
+
+```tsx
+<Alert title={<Alert.Title className={'some-class'} text="some text"></Alert.Title>} />
+```
+
+<!--/GITHUB_BLOCK-->
+
+## Alert message
+
+`message` - The message of an alert should completely explain the alert's content.
+
+## Alert onClose
+
+`onClose` - enabled close button and defined function in click to this button.
```

Comment:
Maybe smth like:
`a callback function that is called when the close button of the alert is clicked. When the this property is defined, a close button is being displayed.`